bitnami/spring-cloud-dataflow/values.yaml

```diff
@@ -0,0 +1,735 @@
+    ##
+    annotations: {}
+
+    ## Enable TLS configuration for the hostname defined at ingress.hostname parameter
+    ## TLS certificates will be retrieved from a TLS secret with name: {{- printf "%s-tls" .Values.ingress.hostname }}
+    ## You can use the ingress.secrets parameter to create this TLS secret or relay on cert-manager to create it
+    ##
+    tls: false
+
+    ## The list of additional hostnames to be covered with this ingress record.
+    ## Most likely the hostname above will be enough, but in the event more hosts are needed, this is an array
+    ## extraHosts:
+    ## - name: dataflow.local
+    ##   path: /
+
+    ## The tls configuration for additional hostnames to be covered with this ingress record.
+    ## see: https://kubernetes.io/docs/concepts/services-networking/ingress/#tls
+    ## extraTls:
+    ## - hosts:
+    ##     - dataflow.local
+    ##   secretName: dataflow.local-tls
+
+    ## If you're providing your own certificates, please use this to add the certificates as secrets
+    ## key and certificate should start with -----BEGIN CERTIFICATE----- or
+    ## -----BEGIN RSA PRIVATE KEY-----
+    ##
+    ## name should line up with a tlsSecret set further up
+    ## If you're using cert-manager, this is unneeded, as it will create the secret for you if it is not set
+    ##
+    ## It is also possible to create and manage the certificates outside of this helm chart
+    ## Please see README.md for more information
+    ##
+    secrets: []
+    ## - name: dataflow.local-tls
+    ##   key:
+    ##   certificate:
+
+  ## Add init containers to the Dataflow Server pods.
+  ## Example:
+  ## initContainers:
+  ##   - name: your-image-name
+  ##     image: your-image
+  ##     imagePullPolicy: Always
+  ##     ports:
+  ##       - name: portname
+  ##         containerPort: 1234
+  ##
+  initContainers: {}
+
+  ## Add sidecars to the Dataflow Server pods.
+  ## Example:
+  ## sidecars:
+  ##   - name: your-image-name
+  ##     image: your-image
+  ##     imagePullPolicy: Always
+  ##     ports:
+  ##       - name: portname
+  ##         containerPort: 1234
+  ##
+  sidecars: {}
+
+  ## Dataflow Server Pod Disruption Budget configuration
+  ## ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb/
+  ##
+  pdb:
+    create: false
+    ## Min number of pods that must still be available after the eviction
+    ##
+    minAvailable: 1
+    ## Max number of pods that can be unavailable after the eviction
+    ##
+    # maxUnavailable: 1
+
+  ## Dataflow Server Autoscaling parameters.
+  ##
+  autoscaling:
+    enabled: false
+    # minReplicas: 1
+    # maxReplicas: 11
+    # targetCPU: 50
+    # targetMemory: 50
+
+## Spring Cloud Skipper parameters.
+##
+skipper:
+  ## Set to true to enable Spring Cloud Skipper component.
+  ## Note: it'll be also enabled if streams are enabled in Dataflow server configuration.
+  ##
+  enabled: true
```

Comment:
This could cause conflict with `server.configuration.streamingEnabled`. Since `skipper` is required if streaming is enabled I would keep them as only one parameter (`server.configuration.streamingEnabled`) and make the rest of the logic, which now depends on `skipper.enabled`, depends on `server.configuration.streamingEnabled`.

So if we want to enable streaming then skipper will be enabled and if we want to disable streaming skipper will be disabled.

Reply:
I agree and disagree. I mean, it makes no sense to disable Skipper if `server.configuration.streamingEnabled=true`. However, you may want to enable Skipper even if `server.configuration.streamingEnabled=false` (I cannot think of any use case for this at this very moment, but let's imagine someone wants to do so).
So my proposal is to create a validation in the `_helpers.tpl` to detect the 1st scenario and show an error.

Reply:
Thinking about this again.. Let's imagine that you already have Skipper installed in your cluster, you might want to enable streaming without enabling Skipper (since you already have it).

Reply:
Yes, I agree with the second scenario and the proposal sounds good to me.

Reply:
Then we would need a value to provide the URL, right? something like `externalSkipper.url`

Reply:
Yep, let me fix that